src/request/iframe/index.html

```diff
@@ -14,6 +14,7 @@
         <script defer bundle-common src="../../lib/CookieJar.js"></script>
         <script defer bundle-common src="../../lib/BrowserDetection.js"></script>
         <script defer bundle-common src="../../lib/AccountStore.js"></script>
+        <script defer bundle-common src="../../lib/Constants.js"></script>
```

Comment:
I think, since the invalid-network-config error can happen now in ALL requests, the Constants.js needs to be added to all requests' index.html as well... sorry :P

Reply:
Should be already there everywhere :)